third_party/silabs/efr32_sdk.gni

```diff
@@ -65,6 +68,11 @@ declare_args() {
   sl_openthread_root = "${efr32_sdk_root}/util/third_party/openthread"
 }
 
+if (enable_openthread_cli == -1) {
+  # enable by default for thread/non-wifi-ncp builds
+  enable_openthread_cli = !(use_rs9116 || use_wf200 || use_SiWx917)
```

Comment:
Is there a reason we couldn't use that expression directly in the `declare_args` instead of doing the -1 thing?

Reply:
Ok, in that case use enum-style like "auto", "enabled", "disabled" as strings. Do not use -1 for something that should be a bool (I believe gn generally understands types)